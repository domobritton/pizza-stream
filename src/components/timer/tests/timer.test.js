import React from 'react';
import renderer from 'react-test-renderer';
import Timer from '../';

describe('Timer', () => {
  test('snapshot-1', () => {
    expect(
      renderer
        .create(<Timer timer={60} />)
    ).toMatchSnapshot();
  });
  test('snapshot-2', () => {
    expect(
      renderer
        .create(<Timer timer={0} />)
    ).toMatchSnapshot();
  });
});