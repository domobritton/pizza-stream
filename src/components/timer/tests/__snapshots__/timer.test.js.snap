// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer snapshot-1 1`] = `
<section
  className="css-qwrdz"
>
  <div
    className="css-1b79e8j"
  >
    It'll take about
  </div>
  <div
    className="css-1jz0aa"
  >
    60 minutes
  </div>
</section>
`;

exports[`Timer snapshot-2 1`] = `
<section
  className="css-qwrdz"
>
  <div
    className="css-1b79e8j"
  >
    Your pizza has arrived!
  </div>
  <div
    className="css-1jz0aa"
  >
    0 minutes
  </div>
</section>
`;
